Examples/Scripts/Python/full_chain_odd.py

```diff
@@ -73,19 +74,40 @@
             rnd=rnd,
             outputDirRoot=outputDir,
         )
+    if not g4_simulation:
+        addFatras(
```

Comment:
I wonder if we should create a new helper like `addSimulation` and switch between fatras and g4 there because it is getting a little cumbersome. but that's something for a different PR

Reply:
Fatras and G4 don't use the same argument at that point it would honestly be more cumbersome than helpful I think.

Reply:
but the only difference is the detector argument right now no?

Reply:
One use a G4 detector,  the other use a tracking geometry. For a DD4Hep they look almost the same but for a different formt of detector (gdml, ...) it get more complex.